wiv.js

```diff
@@ -22,45 +22,82 @@ function wiv(params) {
 
   function initWiv(wiv) {
     //style wiv elements
+    wiv.meta = {};
     wiv.style.display = "inline-block";
-    wiv.style.borderRadius = parseFloat(wiv.dataset.wivHeight) + "px";
```

Comment:
@jjkaufman: I removed this line completely because, for some reason that I can't even pretend to understand, calling this on the fly caused the whole page to freeze in Firefox. In fact, setting any style on a wiv in the manner within the `sizeWiv` function caused the page to freeze.

Is this border radius setting necessary? It doesn't seem to make a difference.

Reply:
I don't think so. Early artifact from an idea that didn't make sense (Circle wivs)